source/platforms/github/GitHubAPI.ts

```diff
@@ -93,7 +93,7 @@ export class GitHubAPI {
     return allComments
       .filter(comment => v.includes(comment.body, dangerIDMessage)) // does it contain the right danger ID?
       .filter(comment => comment.user.id === userID) // Does it have the right user ID?
-      .filter(comment => v.includes(comment.body, dangerSignaturePostfix)) // Does it look like a danger message?
+      .filter(comment => v.includes(comment.body, dangerSignature)) // Does it look like a danger message?
```

Comment:
I'm wondering if this filter is even necessary. I'm  not a huge  fan of exporting a different version of the signature just to test if it is present or not, but I didn't want to check the commit ID in here either.